### PR TITLE
refactor(cli): simplify

### PR DESCRIPTION
**Pull request by jrasm91**

### CODE REVIEW

File: e2e/src/cli/specs/login.e2e-spec.ts [hunk 1]
```diff
@@ -21,7 +21,9 @@ describe(`immich login-key`, () => {
 
   it('should require a valid key', async () => {
     const { stderr, exitCode } = await immichCli(['login-key', app, 'immich-is-so-cool']);
-    expect(stderr).toContain('Failed to connect to server http://127.0.0.1:2283/api: Error: 401');
+    expect(stderr).toContain('Failed to connect to server');
+    expect(stderr).toContain('Invalid API key');
+    expect(stderr).toContain('401');
```
Comment (jrasm91) [inline e2e/src/cli/specs/login.e2e-spec.ts:26]: For API errors, it'll log the response body now.

File: cli/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  instanceUrl: string;
+  apiKey: string;
+}
+
+export const authenticate = async (options: BaseOptions): Promise<void> => {
+  const { configDirectory: configDir, instanceUrl, apiKey } = options;
+
+  // provided in command
+  if (instanceUrl && apiKey) {
+    await connect(instanceUrl, apiKey);
```
Comment (danieldietzler) [inline cli/src/utils.ts:22]: I believe you miss a return here, right?

**Review by danieldietzler — APPROVED** [review]
Very easy to read, awesome! :)

File: cli/src/utils.ts (hunk 2 above)
Comment (jrasm91) [reply cli/src/utils.ts:22]: Yuuuuup.

**Pull request merged**